files/image_config/interfaces/interfaces.j2

```diff
@@ -21,7 +21,7 @@ iface lo {{ 'inet' if prefix | ipv4 else 'inet6' }} static
 # The management network interface
 auto eth0
 {% if MGMT_INTERFACE %}
-{% for (name, prefix) in MGMT_INTERFACE %}
+{% for (name, prefix) in MGMT_INTERFACE|sort %}
```

Comment:
why sort put v4 ahead of v6? is that a co-incidence?

Reply:
The name field is the same, eth0, for both v4 and v6. The sort is based on what is taken in the prefix

Reply:
there is no guarantee that v4 prefix will be ahead of v6 prefix.